wandb/sdk/artifacts/artifact.py

```diff
@@ -1999,66 +2000,21 @@ def link(self, target_path: str, aliases: Optional[List[str]] = None) -> None:
         Raises:
             ArtifactNotLoggedError: if the artifact has not been logged
         """
-        self._ensure_logged("link")
         if not self._ttl_is_inherited:
             termwarn(
                 "Artifact TTL will be removed for source artifacts that are linked to portfolios."
             )
-        self._link(target_path, aliases)
-
-    @normalize_exceptions
-    def _link(self, target_path: str, aliases: Optional[List[str]] = None) -> None:
-        if ":" in target_path:
-            raise ValueError(
-                f"target_path {target_path} cannot contain `:` because it is not an "
-                f"alias."
-            )
```

Comment:
We lost this check. I can now do `artifact.link("my_portfolio:my_alias")`, which created a portfolio named `my_portfolio:my_alias`, which then fails to load in the web app because the `:` is not handled correctly.

Reply:
Oh, does run.link_artifact() not have this check?